spec/02-integration/03-admin_api/08-upstreams_routes_spec.lua

```diff
@@ -37,25 +38,26 @@ dao_helpers.for_each_dao(function(kong_config)
 describe("Admin API", function()
   local client
   local config_db
```

Comment:
Ok... Good to know. WE might want to double-check the luacheck settings under the `spec` folder then... 🤔